tests/openadapt/privacy/providers/test_comprehend_scrub.py

```diff
@@ -1,10 +1,20 @@
 """Module to test ComprehendScrubbingProvider."""
 
+from botocore.exceptions import NoRegionError
+import pytest
 
 from openadapt.privacy.providers.aws_comprehend import ComprehendScrubbingProvider
 
 scrub = ComprehendScrubbingProvider()
 
+try:
+    scrub.scrub_text("hello Bob smith")
+except NoRegionError:
+    pytestmark = pytest.mark.skip(
+        reason="AWS Config Files not setup correctly. Please see "
```

Comment:
Can you please define this message in a separate line? e.g.

```
msg = (
    "..."
    "..."
)
```

Reply:
Addressed in: 
aca3db5